pydm/widgets/drawing.py

```diff
@@ -1111,71 +1111,86 @@ def p2d(pt):
             "convert point to drawing coordinates"
             # drawing coordinates are centered: (0,0) is in center
             # our points are absolute: (0,0) is upper-left corner
-            u, v = map(int, pt.split(","))
+            if isinstance(pt, str):
+                # 2022-05-11: needed for backwards compatibility support
+                # PyDM releases up to v1.15.1
+                # adl2pydm tags up to 0.0.2
+                pt = tuple(map(int, pt.split(",")))
+            u, v = pt
             return QPointF(u+x, v+y)
 
         if len(self._points) > 1:
             for i, p1 in enumerate(self._points[:-1]):
                 painter.drawLine(p2d(p1), p2d(self._points[i+1]))
```

Comment:
In retrospect, I wonder if the easiest solution would have been to override the painter to simply draw an extra line.